docs/site/package.json

```diff
@@ -27,6 +27,8 @@
     "object-assign": "^4.1.0",
     "raw-loader": "^0.5.1",
     "react-hot-loader": "^3.0.0-beta.2",
+    "react-motion": "^0.4.5",
```

Comment:
I'm not sure, does material-ui slide transition takes advantage of touchMove events?